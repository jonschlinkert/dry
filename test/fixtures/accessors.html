{% extends "accessors" %}

<!-- block -->

{% block "a" %}
a: "accessors.html"
  {% block "b" %}
  b: "accessors.html" 
  {% endblock %}
{% endblock %}

{% block "a.b" %}
a.b: "accessors.html"
{% endblock %}

<!-- block -->

{% block "c" %}
c: "accessors.html"
{% endblock %}

{% block "c.d" %}
c.d: "accessors.html"
{% endblock %}