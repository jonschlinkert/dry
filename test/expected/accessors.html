

a: "accessors.html"
  
a.b: "accessors.html"


c: "accessors.html"
